gulpfile.js

```diff
@@ -0,0 +1,62 @@
+
+/*
+ * This file is part of the Lug package.
+ *
+ * (c) Eric GELOEN <geloen.eric@gmail.com>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+var gulp = require('gulp');
+var concat = require('gulp-concat');
+var gulpif = require('gulp-if');
+var uglify = require('gulp-uglify');
+var uglifycss = require('gulp-uglifycss');
+
+var prod = process.env.GULP_ENV === 'prod';
+var dest = 'web/assets/';
+
+var paths = {
+    js: [
+        'node_modules/jquery/dist/jquery.min.js',
+        'node_modules/bootstrap/dist/js/bootstrap.min.js',
+        'node_modules/admin-lte/dist/js/app.min.js',
+        'src/Bundle/**/Resources/js/**/*.js' FIXME
+    ],
+    css: [
+        'node_modules/bootstrap/dist/css/bootstrap.min.css',
+        'node_modules/admin-lte/dist/css/AdminLTE.min.css',
+        'node_modules/admin-lte/dist/css/skins/skin-blue.min.css',
+        'node_modules/font-awesome/css/font-awesome.min.css'
+    ],
+    fonts: [
+        'node_modules/font-awesome/fonts/**'
+    ]
+};
+
+gulp.task('js', function () {
```

Comment:
You can have one file per task it's more readable thanks to `wrench`

``` js
var gulp = require('gulp');
var wrench = require('wrench');

/**
 *  This will load all js or coffee files in the gulp directory
 *  in order to load all gulp tasks
 */
wrench.readdirSyncRecursive('./gulp').filter(function(file) {
  return (/\.(js|coffee)$/i).test(file);
}).map(function(file) {
  require('./gulp/' + file);
});

```